<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>734</width>
     <height>452</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Входные данные</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="AlphaLabel">
         <property name="text">
          <string>Сжатие эллипсоида (1/alpha):</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="AEditText"/>
       </item>
       <item row="0" column="1">
        <widget class="QPlainTextEdit" name="NEditText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Поле заполняется автоматически</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ALabel">
         <property name="text">
          <string>Большая полуось эллипсоида (a):</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="NLabel">
         <property name="text">
          <string>Количество пунктов (N):</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPlainTextEdit" name="MuEditText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Поле заполняется автоматически</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="muLabel">
         <property name="text">
          <string>СКО единицы веса (Mu):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="AlphaEditText"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2" colspan="5">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>СКО</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="m_ZLabel">
         <property name="text">
          <string>m_Z (&quot;):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_SLabel">
         <property name="text">
          <string>m_S (m):</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPlainTextEdit" name="REditText"/>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="m_SEditText"/>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="m_AEditText"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="RLabel">
         <property name="text">
          <string>r'':</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPlainTextEdit" name="m_ZEditText"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_ALabel">
         <property name="text">
          <string>m_A (&quot;):</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Управление</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="fileSelectButton">
         <property name="text">
          <string>Загрузить файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mapSelectButton">
         <property name="text">
          <string>Загрузить карту</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="text">
          <string>Вычислить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Статус</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>Вывод</string>
    </property>
    <addaction name="actionPoints"/>
    <addaction name="actionTopoCoord"/>
    <addaction name="actionMatrixNormUrPopr"/>
    <addaction name="actionKorellMatrixOfVector"/>
    <addaction name="actionVectParam"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOutput"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="text">
    <string>Исходные данные</string>
   </property>
  </action>
  <action name="actionTopoCoord">
   <property name="text">
    <string>Топоцентрические координаты</string>
   </property>
  </action>
  <action name="actionMatrixNormUrPopr">
   <property name="text">
    <string>Матрица нормальных уравнений поправок</string>
   </property>
  </action>
  <action name="actionKorellMatrixOfVector">
   <property name="text">
    <string>Корреляционная матрица вектора</string>
   </property>
  </action>
  <action name="actionVectParam">
   <property name="text">
    <string>Вектор необходимых параметров</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
